src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,8 +1,13 @@
-package racingcar.model
+package racingcar.domain
 
 data class Car(
+    val name: String,
     val position: Int = INITIAL_POSITION,
 ) {
+    init {
+        require(name.length <= MINIMUM_NAME_LENGTH) { "자동차 이름은 5자를 초과할 수 없습니다." }
```

Comment:
사소할 수 있지만 이렇게 예외 메시지의 최대 이름 제한 길이도 상수로 주입해준다면 상수 값이 변경되도 따로 예외 메시지를 수정해줄 필요가 없을 것 같아요. 😃

```suggestion
        require(name.length <= MINIMUM_NAME_LENGTH) { "자동차 이름은 ${MINIMUM_NAME_LENGTH}자를 초과할 수 없습니다." }
```